src/main/java/uk/gov/ons/ctp/response/notify/Application.java

```diff
@@ -69,11 +69,4 @@ public static void main(String[] args) {
     log.debug("About to start the Notify Gateway application...");
     SpringApplication.run(Application.class, args);
   }
-
-  @PostConstruct
-  public void initJsonLogging() {
```

Comment:
Is this not used by anything?

Reply:
No, since the stackdriver changes logging is JSON by default